adam/perception/high_level_semantics_situation_to_developmental_primitive_perception.py

```diff
@@ -768,6 +768,7 @@ def _perceive_object(self, situation_object: SituationObject) -> None:
                 # which we will recursively instantiate.
                 perceived_object = self._instantiate_object_schema(
                     only(object_schemata),
+                    {},
```

Comment:
Use the keyword argument format here; otherwise the reader has no clue what this empty dictionary is for